### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by parsaloi** [community/community#52538] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How many labels exists for achievements...

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @Parsaloi 

You can see the complete list in the following link, there it explains how to get them too

https://github.com/drknzz/GitHub-Achievements

**Reply by parsaloi** [reply]
Some  good work on your profile, and you got another Galaxy brain achievement

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
